ui: fix dropdown styles

maxlang opened the pull request:
![screen shot 2016-12-06 at 4 20 33 pm](https://cloud.githubusercontent.com/assets/3691886/20944330/05015570-bbd1-11e6-85a2-208844a2c39f.png)
![screen shot 2016-12-06 at 4 20 14 pm](https://cloud.githubusercontent.com/assets/3691886/20944331/0506b5ec-bbd1-11e6-95d4-16429b25f5c6.png)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/12118)
<!-- Reviewable:end -->

Commit:
ui: fix dropdown styles

This commit changes the css so that the dropdown style matches the
current mocks.

mrtracy commented on pkg/ui/styl/components/select.styl:
```diff
@@ -15,3 +17,13 @@
     display inline-block
     vertical-align middle
     width 200px
+
```
These are not shameful, and they are in the "right place" as such; however, you should probably just move them to the shame file anyway.  That said, please put a comment here that "style overrides for react-select are in shame.styl due to excessive use of !important"

maxlang replied:
Should we just move all of them into `shame.styl`, or alternately a `select-shame.style` file so they're all together but still called out? I don't feel like putting all the select styles into the shame file is using shame correctly. My understanding is that it's more for "quick hacks" and things you intend to fix, but this feels a little more substantial and potentially long term (I don't like it but I don't intend to come back and fix it soon.)

tamird commented:
What is this screenshot? Perhaps you want to include a before/after?

mrtracy commented:
LGTM, I had a good feeling this was possible with css only.

mrtracy commented:
Instead of "before-after" screenshots, maybe just add some information to the commit message about what is being fixed.  Sorry, I didn't notice because I have a lot of context around this change, but its good practice to include more information in the commit message; a sentence would do.

maxlang commented:
@tamird 

Before

![screen shot 2016-12-06 at 4 33 51 pm](https://cloud.githubusercontent.com/assets/3691886/20944528/d4d000bc-bbd1-11e6-8163-1d9d129fda2e.png)
![screen shot 2016-12-06 at 4 33 41 pm](https://cloud.githubusercontent.com/assets/3691886/20944529/d4d70344-bbd1-11e6-863c-6729b1a84fea.png)

Design

![screen shot 2016-12-06 at 4 34 35 pm](https://cloud.githubusercontent.com/assets/3691886/20944562/f138bb90-bbd1-11e6-863b-e9ed7125e0c5.png)

maxlang commented:
Actually I'll push the dropdown bit a bit lower to conform with the design.

tamird commented:
+1 to meaningful commit messages.

On Tue, Dec 6, 2016 at 4:35 PM, Max Lang <notifications@github.com> wrote:

> Actually I'll push the dropdown bit a bit lower to conform with the design.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/12118#issuecomment-265279944>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABdsPJIxlgz1GRyJ-6iT5krrTxDRXKyiks5rFdU7gaJpZM4LF563>
> .
>

maxlang commented:
Changed commit message and fixed dropdown border/shadow/top.

![screen shot 2016-12-07 at 2 45 02 pm](https://cloud.githubusercontent.com/assets/3691886/20985319/03b9a11a-bc92-11e6-8664-6d4fd2f74b6c.png)
![screen shot 2016-12-07 at 2 44 54 pm](https://cloud.githubusercontent.com/assets/3691886/20985320/03bb9830-bc92-11e6-98b4-17fd9071acd3.png)

vivekmenezes commented:
LGTM

kuanluo commented:
We just talked about a few minor tweaks for the dropdown menu to make sure they're polished. @maxlang can you add the notes here to keep track?
